configs/components/rubygem-hocon.rb

```diff
@@ -1,20 +1,22 @@
 component "rubygem-hocon" do |pkg, settings, platform|
-  instance_eval File.read('configs/components/_base-rubygem.rb')
-
   pkg.version "1.2.5"
   pkg.md5sum "e7821d3a731ab617320ccfa4f67f886b"
-  pkg.url "https://rubygems.org/downloads/hocon-#{pkg.get_version}.gem"
-  pkg.mirror "#{settings[:buildsources_url]}/hocon-#{pkg.get_version}.gem"
 
-  install_steps = ["#{settings[:gem_install]} hocon-#{pkg.get_version}.gem"]
+  instance_eval File.read('configs/components/_base-rubygem.rb')
 
   # Overwrite the base rubygem's default GEM_HOME with the vendor gem directory
   # shared by puppet and puppetserver. Fall-back to gem_home for other projects.
   pkg.environment "GEM_HOME", (settings[:puppet_gem_vendor_dir] || settings[:gem_home])
 
+  # For most platforms, the hocon gem has the same installation command as any
+  # other gem. This general install step is no different from the install step
+  # in _base_rubygem (but it must be here because of the special case below):
+  install_steps = ["#{settings[:gem_install]} hocon-#{pkg.get_version}.gem"]
+
+  # Special case for macOS:
   if platform.is_macos?
     # The unicode characters in one of the hocon gem's spec files will make tar
-    # blow up when extracting the runtime tarball on MacOS if not using unicode.
+    # blow up when extracting the runtime tarball on macOS if not using unicode.
     # We don't need the spec files anyway, so remove them:
```

Comment:
Is this global? Not in scope for this PR, but perhaps a different one?

Reply:
I don't really care to do this globally - it's just a brand name in a comment I fixed since I was already there